### PR TITLE
fix wrong results in activity

### LINKED ISSUE #5544: Wrong results in activity
image
https://kodadot.xyz/rmrk2/gallery/12192538-9692fa834afcce2d3e-AC99-MATRYOSHKA_VESTA-00000012

image

- showing unlist under sales does not make sense no?

### PR DESCRIPTION
**Pull request by iMac7**
**Thank you for your contribution** to the [Koda - Generative Art Marketplace](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #5544 
- [ ] Requires deployment <worker/stick/speck>

#### Before submitting pull request, please make sure:

- [ ] My contribution builds **clean without any errors or warnings**
- [ ] I've merged recent default branch -- **main** and I've no conflicts
- [ ] I've tried to respect high code quality standards
- [ ] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ahp/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw&usdamount=undefined)

#### Community participation

- [x] [Are you at Koda Ecosystem Telegram?](https://t.me/koda_eco)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue [hunk 1]
```diff
@@ -66,6 +66,13 @@ const cssActive = (value) => {
     ),
   }
 }
+
+const filteredInteractions = computed(() => {
+  if (interactions.value.length === allInteractions.length) {
+    return interactions.value
+  }
+  return interactions.value.filter((interaction) => interaction !== 'UNLIST')
+})
```
Comment (hassnian) [inline components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:75]: why not just remove `UNLIST` from `allInteractions`
CleanShot 2024-04-01 at 17 07 52@2x

Comment (hassnian) [reply components/gallery/GalleryItemTabsPanel/GalleryItemActivity.vue:75]: From what I can see we are just adding more code when we could just remove UNLIST from that array

if it's needed once again we can add it back, it's not like it's a lot of code , shouldn't be an issue

**Review by hassnian — APPROVED** [review]

**Review by vikiival — APPROVED** [review]

**Comment by prury** [comment]
@iMac7 please resolve conversations

**Comment by prury** [comment]
pay 10 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $10 @ 8.63 USD/DOT ~ 1.159 $DOT
🧗 [14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw](https://polkadot.subscan.io/account/14dLrAvjDw7cFLaxM7VWR6YtHd7bGZY2QpzXDhAJJ4yv6oGw)
🔗 [0xf9a5e9305f47de1e73750bb923224438979cb47eb5ed7e84de553061eaccc65a](https://polkadot.subscan.io/extrinsic/0xf9a5e9305f47de1e73750bb923224438979cb47eb5ed7e84de553061eaccc65a)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**